### DISCUSSION TITLE
When will pokemon sword playable in skyline emulator

### DISCUSSION THREAD

**Discussion by Rakhisinhacitra** [skyline-emu/skyline#273] [Q&A]
When will pokemon sword playable in skyline emulator

**Comment by mlgprorektm8** [comment]
in maybe 20-30 years

**Reply by ghost** [reply]
No in 4 months I believe in it

**Comment by Kingscott1330** [comment]
I actually have faith in these developers for sure my pixel 6 pro is hitting some nice frames already on sonic phones being released now have the hardware to make switch games playable of sure

**Comment by JerryM33** [comment]
6 months is enough

**Reply by ghost** [reply]
I say 8 months for hard-core games
